src/analysis.ts

```diff
@@ -482,3 +446,81 @@ export async function analyzeGit(options: GitOptions, requestOptions?: RequestOp
 
   return result;
 }
+
+interface CreateBundleFromFolders extends FolderOptions {
+  supportedFiles?: ISupportedFiles;
+  baseDir?: string;
+  fileIgnores?: string[];
+}
+
+let createBundleFromFoldersDefaults = {
+  baseURL: defaultBaseURL,
+  sessionToken: '',
+  symlinksEnabled: false,
+  maxPayload: MAX_PAYLOAD,
+  defaultFileIgnores: IGNORES_DEFAULT,
+  source: '',
+};
```

Comment:
nit: I would put these defaults inline on L472, since the var is not used anywhere else I don't see the point of declaring it separately in the file scope.

Reply:
It's still outside, but now a const, I meant just putting those properties directly in the method where they're used, like:
```
export async function createBundleFromFolders(options: CreateBundleFromFolders): Promise<RemoteBundle | null> {
  const analysisOptions = {
    baseURL: defaultBaseURL,
    sessionToken: '',
    symlinksEnabled: false,
    maxPayload: MAX_PAYLOAD,
    defaultFileIgnores: IGNORES_DEFAULT,
    source: '',
    ...options,
  };

Reply:
As I mentioned, I have extracted it and reused in multiple methods as I noticed same things are being repeated.